src/graphql/blocks.ts

```diff
@@ -1,13 +1,36 @@
 import snapshot from '@snapshot-labs/snapshot.js';
 import networks from '@snapshot-labs/snapshot.js/src/networks.json';
 import redis from '../redis';
+import fetch from 'cross-fetch';
```

Comment:
External dependencies should appear first
```suggestion
import fetch from 'cross-fetch';
import snapshot from '@snapshot-labs/snapshot.js';
import networks from '@snapshot-labs/snapshot.js/src/networks.json';
import redis from '../redis';
```